### DISCUSSION TITLE
DiscordBot v2.0.1

### DISCUSSION THREAD

**Discussion by JaxkDev** [JaxkDev/DiscordBot#36] [Releases]
# DiscordBot v2.0.1

DiscordBot brings a powerful promise API to interact with [Discord](https://discord.com) by a bot.

**PLEASE NOTE THIS IS NOT A FUNCTIONAL CHAT BRIDGE PLUGIN** This is a separate plugin which will be released very shortly. (*watch this space*)

# Changelog

A patch update changing minor things, if your plugin works with v2.0.0 it will work with this update no changes required.

- Embed description character limit has been doubled as updated by discord on 01/07/2021

- Added support for Nitro users to allow message content of up to 4000 characters.

- Corrected composer package type to 'library'

- Added an emergency notice when privileged intents have not been enabled on the discord developer dashboard.

### API Changes
None

### Model Changes:
  - Message
    - Content limit adjusted to 4000 characters for nitro users.
  - Embed Description
    - Character limit has been doubled.

### Event Changes:
None

### Storage Changes:
None

### Command Changes
None

<PHAR COMING SOON>

<hr /><em>This discussion was created from the release <a href='https://github.com/DiscordBot-PMMP/DiscordBot/releases/tag/2.0.1'>DiscordBot v2.0.1</a>.</em>

**Comment by DretooFuni** [comment]
You should create a demo api plugin

**Reply by JaxkDev** [reply]
The demo for the API is in the API.md
In terms of plugins https://GitHub.com/DiscordBot-pmmp/ChatBridge is the next best demo once completed/released
